squad/api/rest.py

```diff
@@ -1441,6 +1441,13 @@ class Meta:
         fields = '__all__'
 
 
+class TestJobUpdateSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
Could you please propagate read-only fields:
```python
id = serializers.IntegerField(read_only=True)
external_url = serializers.CharField(source='url', read_only=True)
```

Reply:
Yeah, makes sense. This makes the API consistent.